gst/nnstreamer/tensor_common.c

```diff
@@ -408,11 +407,15 @@ gst_tensor_config_from_video_info (GstTensorConfig * config,
   config->info.dimension[1] = width;
   config->info.dimension[2] = height;
   config->info.dimension[3] = 1; /** Supposed 1 frame in tensor, change this if tensor contains N frames. */
-
+#if 0
+  /**
+   * @todo To fix coverity issue, now block these lines.
+   * If NNS_TENSOR_RANK_LIMIT is larger than 4, unblock these to initialize the tensor dimension.
+   */
   for (i = 4; i < NNS_TENSOR_RANK_LIMIT; i++) {
```

Comment:
It seems that you must update the exisitng for statement because you removed `gint i`. Please check it. 

* before: i =4; 
* after gint =4;